.devcontainer/README.md

```diff
@@ -53,6 +53,22 @@ Make sure HF_HOME is sourced in your .bashrc or .zshenv and your vscode default
 
 If `post-create.sh` fails, you can try to debug or [submit](https://github.com/ai-dynamo/dynamo/issues) an issue on github.
 
+## Development Flow
+
+If you make changes to rust code and you want to compile, use [cargo build](https://doc.rust-lang.org/cargo/commands/cargo-build.html). This will update rust binaires such as dynamo-run.
```

Comment:
```suggestion
If you make changes to rust code and you want to compile, use [cargo build](https://doc.rust-lang.org/cargo/commands/cargo-build.html). This will update rust binaries such as dynamo-run.
```